src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLLegImpl.java

```diff
@@ -218,4 +222,35 @@ public DataFetcher<Boolean> walkingBike() {
   private Leg getSource(DataFetchingEnvironment environment) {
     return environment.getSource();
   }
+
+  @Override
+  public DataFetcher<Iterable<Leg>> nextLegs() {
+    return environment -> {
+      if (environment.getSource() instanceof ScheduledTransitLeg) {
+        int numberOfLegs = environment.getArgumentOrDefault("numberOfLegs", 2);
```

Comment:
It's better to set the default in the schema

Reply:
It was decided later on to remove the default argument value, so marking this as resolved